code/mob/living/carbon/human.dm

```diff
@@ -3226,7 +3226,28 @@
 	if (head)
 		worn += head.name
 	else
-		worn += "hair"
+		if (istype(src:mutantrace, /datum/mutantrace/lizard))
+			worn += "scales"
+		else if (istype(src:mutantrace, /datum/mutantrace/werewolf) || istype(src:mutantrace, /datum/mutantrace/pug) || (ismonkey(src)))
+			worn += "fur"
+		else if (istype(src:mutantrace, /datum/mutantrace/cow))
+			worn += "hooves"
+			worn += "fur"
+		else if (istype(src:mutantrace, /datum/mutantrace/roach))
```

Comment:
why the `:`?

Reply:
because I copy/pasted that part and didn't notice it :sheltersweat: will fix